### DISCUSSION TITLE
uncommented const int in u8g2

### DISCUSSION THREAD

**Discussion by Gudzys** [olikraus/u8g2#2037] [Q&A]
The Hitachi LCD 16x2 const int is like this:
const int rs = 12, en = 11, d4 = 5, d5 = 4, d6 = 3, d7 = 2;
LiquidCrystal lcd(rs, en, d4, d5, d6, d7);

How should the uncommented const int look like in u8g2?
u8g2(U8G2_R0, 13, 11, 10, 9, 8);

(//Const int clock=13, data=11, cs=10, rs=9, reset=8, , , ;) 
(//U8G2_ST75256_JLX240160_F_4W_SW_SPI)

**Comment by olikraus** [comment]
Not sure whether I understand your question, but you just need to replace the pin numbers with your pin numbers within your project.

**Comment by Gudzys** [comment]
If you're using the same pin numbers do you do anything else except remove the initial "//" to uncomment the line?
U8G2_ST75256_JLX240160_F_4W_SW_SPI u8g2(U8G2_R0, /* clock=*/ 13, /* data=*/ 11, /* cs=*/ 10, /* dc=*/ 9, /* reset=*/ 8);
 
Thank you again for your assistance.

**Reply by olikraus** [reply]
No, removal of the // is sufficient in this case.

**Comment by Gudzys** [comment]
Step #1: connect the LCD pins to the Arduino pins as described above. Step #2: Uncomment the correct line in the sketch by removing the initial "//" if the Arduino pins are the same. What am I missing?

Thank you. 

Sent with [Proton Mail](https://proton.me/) secure email.

------- Original Message -------

**Reply by olikraus** [reply]
Update pin numbers if required ☺️
